bitlbee does not build

**Describe the bug**
Bitlbee does not build on master.

**To Reproduce**
Steps to reproduce the behavior:
1. `nix-build --no-build-output -A bitlbee`

```
these derivations will be built:
  /nix/store/2a69lfr8r6zh6d20nwvqyxcd6p9yii05-bitlbee-3.6.drv
building '/nix/store/2a69lfr8r6zh6d20nwvqyxcd6p9yii05-bitlbee-3.6.drv'...
builder for '/nix/store/2a69lfr8r6zh6d20nwvqyxcd6p9yii05-bitlbee-3.6.drv' failed with exit code 2; last 10 log lines:
              ^~~~~~~~~~~~~~~~~
  check_util.c:27:12: warning: 'test_strip_newlines' defined but not used [-Wunused-variable]
   START_TEST(test_strip_newlines)
              ^~~~~~~~~~~~~~~~~~~
  check_util.c:11:12: warning: 'test_strip_linefeed' defined but not used [-Wunused-variable]
   START_TEST(test_strip_linefeed){
              ^~~~~~~~~~~~~~~~~~~
  make[1]: *** [Makefile:27: check_util.o] Error 1
  make[1]: Leaving directory '/build/bitlbee-3.6/tests'
  make: *** [Makefile:65: check] Error 2
error: build of '/nix/store/2a69lfr8r6zh6d20nwvqyxcd6p9yii05-bitlbee-3.6.drv' failed
```

**Expected behavior**
The bitlbee derivation should build without errors.

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.3.11, NixOS, 20.03.git.6be20178f08 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - nixpkgs: 25fc378642e

Maintainer information:
```yaml
attribute:
- bitlbee
- bitlbee-steam
- bitlbee-mastodon
- bitlbee-facebook
- bitlbee-discord
# a list of nixos modules affected by the problem
module:
- services/networking/bitlbee
```

@pSub

--- a/pkgs/applications/networking/instant-messengers/bitlbee/default.nix
+++ b/pkgs/applications/networking/instant-messengers/bitlbee/default.nix
@@ -1,4 +1,4 @@
-{ fetchurl, stdenv, gnutls, glib, pkgconfig, check, libotr, python
+{ fetchurl, fetchpatch, stdenv, gnutls, glib, pkgconfig, check, libotr, python
 , enableLibPurple ? false, pidgin ? null
 , enablePam ? false, pam ? null
 }:
@@ -24,6 +24,14 @@ stdenv.mkDerivation rec {
     "--pidfile=/var/lib/bitlbee/bitlbee.pid"
   ] ++ optional enableLibPurple "--purple=1"
     ++ optional enablePam "--pam=1";
+
+  patches = [
+    # This should be dropped once the issue is fixed upstream.
+    (fetchpatch {
+      url = "https://github.com/bitlbee/bitlbee/commit/6ff651b3ec93e5fd74f80766d5e9714d963137bc.diff";
+      sha256 = "144dpm4kq7c268fpww1q3n88ayg068n73fbabr5arh1zryw48qfv";
+    })
+  ];
 
   installTargets = [ "install" "install-dev" ];
 

